ReactWindows/Playground/AppReactPage.cs

```diff
@@ -11,15 +11,15 @@ public override string MainComponentName
         {
             get
             {
-                return "UIExplorerApp";
+                return "Playground";
```

Comment:
See reply below.

---
In reply to: [70992562](https://github.com/ReactWindows/react-native-windows/pull/526#discussion_r70992562) [](ancestors = 70992562)